specs/deneb/p2p-interface.md

```diff
@@ -300,6 +303,11 @@ Clients MUST support requesting sidecars since `minimum_request_epoch`, where `m
 Clients MUST respond with at least one sidecar, if they have it.
 Clients MAY limit the number of blocks and sidecars in the response.
 
+Clients SHOULD consider including a sidecar in the response as soon as it passes the gossip validation rules.
+Clients MUST NOT respond with sidecars that failed gossip validation.
+Clients MUST NOT respond with sidecars related to blocks that failed `fork_choice.on_block`.
+Clients MAY continue serving blocks\sidecars after failed `fork_choice.on_block` if and only if the failure reason is due to missing blobs. (i.e. some sidecars are missing but the available ones have been fully verified against the fully validated block).
```

Comment:
I think this statement is worth to be duplicated in the `BeaconBlocksByRoot` section